src/main/java/mate/academy/springboot/swagger/dto/ProductRequestDto.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.springboot.swagger.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
```

Comment:
I think it would be enough to have getters and setters in a dto

JFYI after java 14 you can use records for such purpose